README.md

```diff
@@ -55,57 +53,79 @@ Here are the examples,
 ```js
 For all blog list,
 
-  <BlogList :showDrafts="true" />
+  <BlogList
+      :showDrafts="true"
+      :posts="posts"
+      :featurePosts="featurePosts"
+    />
 
 For tag list,
-  <TagList :slug="slug" @notfound="<your-callback>" :showDrafts="true"  />
+  <TagList :slug="slug"
+      @notfound="<your-callback>"
+      :showDrafts="true"
+      :tagName:"tagname" />
 
 
 For Blog details,
   <BlogDetail
       :slug="slug"
       @notfound="<your-callback>"
-      :showDrafts="true"
-      :iframely-key="IFRAMELY_KEY"
+      :post="post"
       :recaptcha-key="RECAPTCHA_SITE_KEY"
+      :status="status"
+
+
+      :recommandedPosts="recommandedPosts"
+      :websiteUrl="websiteUrl"
     />
 
 ```
+
+Here `websiteUrl` is the link of the user website to share the links of blog on different platforms.
```

Comment:
Explain all the properties in detail. You can refer [web-file-upload](https://github.com/canopas/web-file-upload)